app/models/clinic.rb

```diff
@@ -17,13 +17,33 @@ class Clinic
   field :active, type: Boolean, default: true
   field :accepts_naf, type: Boolean, default: false
   field :gestational_limit, type: Integer
-  field :costs_9wks, type: Integer
+  field :costs_05wks, type: Integer
```

Comment:
I think for ease of translation, can we strip the leading zeroes here? (Unless there's a reason for it I'm forgetting, in which case let's document it here!)